cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -2093,6 +2093,16 @@ spec:
                         type: string
                       startTime:
                         type: string
+            quotas:
+              type: object
+              properties:
```

Comment:
I think `properties` should go after `nullable`

Reply:
Fixed. Looks like there must be something else though. I'll poke around.

Reply:
Moving quotas to the v1 crd in helm did the trick. Thanks.